### PR TITLE
v6.0.0-beta.0

### PR DESCRIPTION
**Pull request by paulirish**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,3 +1,252 @@
+
+ <a name="6.0.0-beta.0"></a>
+# 6.0.0-beta.0 (2020-03-11)
+[Full Changelog](https://github.com/GoogleChrome/lighthouse/compare/v5.6.0...v6.0.0-beta.0)
+
+(A proper thanks for our new contributors will be in the final 6.0 release notes.)
+
+This beta release will not ship in Chrome DevTools, but we'll ship the final 6.0 release.
```
Comment (addyosmani) [inline changelog.md]: Do we also want to call out that the beta won't be shipped to the PSI front-end for now?

File: changelog.md [hunk 2]
```diff
@@ -1,3 +1,252 @@
...
+
+## Core
+
+* artifacts: add property attribute to MetaElements ([#9978](https://github.com/GoogleChrome/lighthouse/pull/9978))
+* audits: add handling of 'incomplete' results from axe-core ([#10072](https://github.com/GoogleChrome/lighthouse/pull/10072))
+* audits: Add more keywords to blocklist ([#9986](https://github.com/GoogleChrome/lighthouse/pull/9986))
+* config: add loadFailureMode warn ([#9987](https://github.com/GoogleChrome/lighthouse/pull/9987))
+* config: enable unused-javascript by default ([#9854](https://github.com/GoogleChrome/lighthouse/pull/9854))
+* duplicated-javascript: new audit ([ea9d226](https://github.com/GoogleChrome/lighthouse/commit/ea9d226))
+* emulation: bump emulated chrome UA to m80 ([#9967](https://github.com/GoogleChrome/lighthouse/pull/9967))
```
Comment (connorjclark) [inline changelog.md]: we're going to want to give these commit notes the same treatment we do in all other releases, but we can defer that to the actual 6.0 notes.

File: changelog.md [hunk 3]
```diff
@@ -1,3 +1,252 @@
+
+ <a name="6.0.0-beta.0"></a>
+# 6.0.0-beta.0 (2020-03-11)
```
Comment (connorjclark) [inline changelog.md]: aside: I'd want to totally delete the beta notes when 6.0 is out. just noisy :)

**Comment by paulirish** [comment]
This is published on NPM with the `next` tag.

You can install the 6.0 beta via `npm install -g lighthouse@next`

The unedited changelog is here: https://github.com/GoogleChrome/lighthouse/blob/v6.0.0-beta.0/changelog.md#600-beta0-2020-03-11

**Pull request merged**